testbook/testbook.py

```diff
@@ -1,5 +1,11 @@
 import nbformat
 
+try:
+    from pytest import fixture
+except ImportError:
+    def fixture(func, *args, **kwargs):
+        return func
```

Comment:
Maybe we should we add `pytest` into list of dependencies?

Reply:
I'm fine with that. But as it was not a dependency, I got the impression you want it to be independent of the unit test framework.

I'll simplify the code and add explicit dependency on pytest

Reply:
We actually do want testbook to be independent of the unit test framework. I guess we could leave it this way, you need not add an explicit dependency on pytest. Sorry for the confusion.